girder_worker/plugins/docker/tests/docker_integration_test.py

```diff
@@ -0,0 +1,278 @@
+import ConfigParser
+import girder_worker
+import os
+import shutil
+import six
+import stat
+import sys
+import unittest
+
+from girder_worker.core import run, io
+
+test_image = 'cjh1/girder_worker:test'
```

Comment:
Let's get this moved into the Girder org as a precondition for merging.

@zachmullen Can you give @cjh1 perms to the Girder DockerHub org or else migrate that image in?